src/analysis/ContextBar.test.ts

```diff
@@ -32,6 +45,16 @@ jest.mock('src/analysis/utils/cost-utils', () => ({
   runtimeConfigCost: jest.fn(),
 }));
 
+jest.mock('src/analysis/utils/app-utils', () => ({
```

Comment:
Type safety for mocked modules could be improved with:

```suggestion
type AppUtilsExports = typeof import('src/analysis/utils/app-utils');
jest.mock('src/analysis/utils/app-utils', (): AppUtilsExports => ({
```

Same for other mocks.